source/frontend/assets/components/RecipeCard.js

```diff
@@ -111,8 +128,11 @@ class RecipeCard extends HTMLElement {
     elem.appendChild(rating);
 
     const time = document.createElement('time');
-
-    time.innerHTML = `${Math.floor(data.totalTime / 60)} hrs ${data.totalTime % 60} min`;
+    if (data.totalTime >= 60) {
```

Comment:
`>==`

Reply:
when i do >== it gives a syntax error. is this suppose to happen?

Reply:
Actually yes... Unfortunately Javascript doesn't really want to avoid type coercion on >= or <= like how it does on == and !=. Therefore, it's actually my mistake that js doesn't support strict less than or equals to and greater than or equals operators. That's actually the reason that many people actually prefer typescript over javascript. 

So for now we should probably just leave as it is.